features/order/managing_orders/placing_an_order_with_shipment.feature

```diff
@@ -0,0 +1,21 @@
+@managing_orders
+Feature: Shipments are in the state "ready" after checkout
```

Comment:
Feature description should be less technical and as near to business language as possible e.g. `Shipments should be ready to send after checkout`